sql/1.107/contributor_activity.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE contributor_activity (
+    id serial NOT NULL PRIMARY KEY,
+    github_id text NOT NULL UNIQUE,
```

Comment:
`github_id` - yes it should be unique. does it matter that it could be an number string or a SHA mixed character string for commits (ex: 7638417db6d59f3c431d3e1f261cc637155684cd)